packages/overlay/src/integrations/sentry/components/explore/ExploreTabDetails.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { Navigate, Outlet, Route, Routes } from 'react-router-dom';
+import Tabs from '~/components/Tabs';
+import sentryDataCache from '../../data/sentryDataCache';
+import DeveloperInfoTab from '../../tabs/DeveloperInfo';
+import SdksTab from '../../tabs/SdksTab';
+import TracesTab from '../../tabs/TracesTab';
+
+export default function ExploreTabDetails() {
+  const [activeTab, setActiveTab] = useState('');
+
+  const localTraces = sentryDataCache.getTraces().filter(t => sentryDataCache.isTraceLocal(t.trace_id) !== false);
+
+  const tabs = [
+    {
+      id: 'traces',
+      title: 'Traces',
+      active: activeTab === 'traces',
+      onSelect: () => setActiveTab('traces'),
+      notificationCount: {
+        count: localTraces.length,
+      },
+    },
+    {
+      id: 'sdks',
+      title: 'SDKs',
+      active: activeTab === 'sdks',
+      onSelect: () => setActiveTab('sdks'),
+    },
+    {
+      id: 'devInfo',
+      title: 'Developer Info',
```

Comment:
Shall we name this "Raw Events" or something now?

Reply:
yeah we can rename this to "Raw Events" OR "Events" OR "All Events"
Lets go with "Raw  Events" for now.